src/ggrc-client/js/plugins/utils/models-utils.js

```diff
@@ -158,6 +159,32 @@ function getCustomAttributableModels() { // eslint-disable-line
     .map((modelName) => businessModels[modelName]);
 }
 
+
+async function initAuditTitle(instance, isNewInstnace) {
+  const isAuditTitle = (
+    instance.constructor
+    && instance.constructor.model_singular === 'Audit'
+  );
+  const hasNoNeedToInitTitle = !(
```

Comment:
Maybe we should simplify it to remove `not` part? Personally to me such conditionals look complex
```suggestion
  const hasNeedToInitTitle = (
```
And then we need to change `if` condition below: put everything inside if block
```
if (hasNeedToInitTitle){
// all logic below
}
```